lib/diagnostic.rb

```diff
@@ -5,36 +5,41 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    max = 0
+    list.each { |item|
+      if item.length > max.length
```

Comment:
Yes this would work. 

But remember the spaceship comparison

`list.max { |a , b| a.length <=> b.length }`